packages/wrangler/src/__tests__/tail.test.ts

```diff
@@ -166,24 +166,48 @@ describe("tail", () => {
 
   /* Printing */
 
-  it("logs incoming messages", async () => {
+  it("logs incoming messages in JSON format", async () => {
     await runWrangler("tail test-worker --format json");
 
+    /* request event */
     const requestEvent = generateMockRequestEvent();
-    const requestMessage = serialize(
-      generateMockEventMessage({ event: requestEvent })
-    );
-    api.ws.send(requestMessage);
-    expect(std.out).toMatch(deserializeToJson(requestMessage));
+    const requestMessage = generateMockEventMessage({ event: requestEvent });
+    const serializedRequestMessage = serialize(requestMessage);
+
+    api.ws.send(serializedRequestMessage);
+    expect(std.out).toMatch(deserializeToJson(serializedRequestMessage));
 
+    /* scheduled event */
     const scheduledEvent = generateMockScheduledEvent();
-    const scheduledMessage = serialize(
-      generateMockEventMessage({
-        event: scheduledEvent,
-      })
-    );
-    api.ws.send(scheduledMessage);
-    expect(std.out).toMatch(deserializeToJson(scheduledMessage));
+    const scheduledMessage = generateMockEventMessage({
+      event: scheduledEvent,
+    });
+    const serializedEventMessage = serialize(scheduledMessage);
+
+    api.ws.send(serializedEventMessage);
+    expect(std.out).toMatch(deserializeToJson(serializedEventMessage));
+  });
+
+  it("logs messages in pretty-printing format", async () => {
+    await runWrangler("tail test-worker --format pretty");
+
+    /* request event */
+    const requestEvent = generateMockRequestEvent();
+    const requestMessage = generateMockEventMessage({ event: requestEvent });
+    const serializedRequestMessage = serialize(requestMessage);
+
+    api.ws.send(serializedRequestMessage);
+    expect(std.out).not.toMatch(deserializeToJson(serializedRequestMessage));
+
+    /* scheduled event */
+    const scheduledEvent = generateMockScheduledEvent();
+    const scheduledMessage = generateMockEventMessage({
+      event: scheduledEvent,
+    });
+    const serializedEventMessage = serialize(scheduledMessage);
+
+    api.ws.send(serializedEventMessage);
+    expect(std.out).not.toMatch(deserializeToJson(serializedEventMessage));
```

Comment:
How about we use a `toMatchInlineSnapshot()` check here?

Reply:
I don't know how that works 😞  I'll google rq

Reply:
oh neat! yeah sure

Reply:
i'll send a follow-up PR because I also wound up splitting the test file into different sub-suites a-la KV to make the test output more easily readable at a glance